### DISCUSSION TITLE
Docker版本固件和无Docker版本固件可以互刷吗？

### DISCUSSION THREAD

**Discussion by YuMXuan** [coolsnowwolf/lede#9589] [R68S / 电犀牛专区] [answered]
新人小白，冒昧问一下Docker版本固件和无Docker版本固件可以互刷吗？
网页上刷写新的固件是个什么原理？不选择保留配置是不是就等于说是刷写了一遍纯净的固件，是不是就不会留下之前固件的东西？
我发现就算是选择了保留配置，自己上传安装的ipk也会消失，这样会不会留下残留文件？

**Comment by npcXxX** [comment, marked as answer]
lean大的都可以互刷的，不保留就是新系统

**Reply by YuMXuan** [reply]
感谢
